src/PlusServer/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ SET(${PROJECT_NAME}_CMD_SRCS
   Commands/vtkPlusGetUsParameterCommand.cxx
   Commands/vtkPlusAddRecordingDeviceCommand.cxx
   Commands/vtkPlusGenericSerialCommand.cxx
+  Commands/vtkPlusWinProbeCommand.cxx
```

Comment:
There needs to be these type changes as are done for `PLUS_USE_ATRACSYS` and `PLUS_USE_STEALTHLINK` farther down the file
```
IF(PLUS_USE_WINPROBE_VIDEO)
  LIST(APPEND ${PROJECT_NAME}_SRCS Commands/vtkPlusWinProbeCommand.cxx)
  LIST(APPEND ${PROJECT_NAME}_HDRS Commands/vtkPlusWinProbeCommand.h)
ENDIF()
```